src/components/Main/index.jsx

```diff
@@ -25,12 +24,15 @@ const selectorGenerator = new CssSelectorGenerator({
   selectors: ['id', 'tag', 'nthchild'],
 });
 
-const getWrappedText = (node) => {
-  const textNodes = filter(
-    node.childNodes, child => child.nodeType === child.TEXT_NODE);
-  const combined = trim(
-    textNodes.map(textNode => textNode.textContent).join(''));
-  return combined || null;
+const getWrappedText = (node, rich = false) => {
```

Comment:
That's reasonable (though won't make any actual difference since javascript won't enforce the args)